### DISCUSSION TITLE
Bulk config re: modified AdGuardHome.yaml

### DISCUSSION THREAD

**Discussion by cyb3rz3us** [AdguardTeam/AdGuardHome#4558] [Q&A]
As a means by which to import several clients into AdGuard, I simply stopped the Docker container and then modified the `AdGuardHome.yaml` file.  Aside from one minor issue, this crude import seemed to work fine.  Is this an acceptable means by which to make changes to AdGuard?

I'm basically looking for any gotchas that may result from this method of configuration.  The issue I mentioned above had to do with the per client **Block specific services** config not taking effect.  This was because the **Use global blocked services option** in the GUI was selected even though I had the `use_global_settings: false` set for the same client in the actual YAML file.

**Comment by ainar-g** [comment]
Sorry for the late response.  As long as the new configuration is valid, there shouldn't be any issues with this.

The (non-)usage of global blocked services by a client is controlled by the separate `use_global_blocked_services` boolean.  This part of the configuration file is unfortunately not very well documented, and we plan on fixing that.
